lib/kitchen/version.rb

```diff
@@ -17,5 +17,5 @@
 # limitations under the License.
 
 module Kitchen
-  VERSION = "1.18.0".freeze
+  VERSION = "1.19.0".freeze
```

Comment:
you'll want to remove this.

Reply:
ok, rebased on master and now its showing no changes. is there a bot that increments it or something?